src/Middleware/ProfileMiddleware.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace EightPoints\Bundle\GuzzleBundle\Middleware;
+
+use Symfony\Component\Stopwatch\Stopwatch;
+
+class ProfileMiddleware
+{
+    /**
+     * @var Stopwatch
+     */
+    private $stopwatch;
+
+    /**
+     * ProfileMiddleware constructor.
```

Comment:
`ProfileMiddleware constructor.` - please remove this comment line

Reply:
done